specs/latest/1.0/index.html

```diff
@@ -919,6 +926,50 @@ <h4>Context creation parameters</h4>
                 </p>
               </div>
             </dd>
+        <dt><span class=prop-value>pixelFormat</span></dt>
+            <dd>
+              <p>
+                If defined, require a backbuffer of a specific effective internal format.
+                This is a requirement, and if the requested format is not availible, context creation will fail.
+              </p>
+              <p>
+                The allowed values are:
+              </p>
+              <dl><dd>
+                  <p>
+                    <em>undefined</em>
+                  </p>
+                  <p>
+                    If not defined, choose implicitly based on <code>alpha</code>.
+                    This is the historical behavior.
+                  </p>
+                  <p>
+                    <em>RGBA8</em>
+                  </p>
+                  <p>
+                    Explicitly require an RGBA8 backbuffer.
+                    This is similar to <code>{alpha: true, pixelFormat: "implicit"}</code>.
+                    This is the same effective format as a renderbuffer or texture specified with a <code>sizedFormat</code> or <code>internalFormat</code> of <code>RGBA8</code>,
+                    or a texture specified with <code>{internalFormat: RGBA, unpackFormat: RGBA, unpackType: UNSIGNED_BYTE}</code>.
+                  </p>
+                  <p>
+                    <em>SRGB8_ALPHA8</em>
+                  </p>
+                  <p>
+                    Explicitly require an SRGB8_ALPHA8 "sRGB-encoded" backbuffer.
```

Comment:
In WebGL 1.0 it would add a lot of unnecessary validation code to expose this functionality without auto-enabling the EXT_sRGB extension, if it's supported. If we're moving forward with exposing this functionality on WebGL 1.0 then I'd like using this pixel format as being defined as enabling that extension.